.dependabot/config.yml

```diff
@@ -0,0 +1,12 @@
+version: 1
+update_configs:
+  - package_manager: "java:maven"
+    directory: "/bom/runtime"
+    update_schedule: "daily"
+    allowed_updates:
+      - match:
+          update_type: "security"
+      - match:
+          dependency_name: "org.flywaydb:flyway-core"
+      - match:
+          dependency_name: "org.eclipse.jgit:org.eclipse.jgit"
```

Comment:
With this config, it will just create PRs?

Reply:
Yes, automerged updates must be enabled at the account level (from account settings in your dashboard) before they can be configured on a project.

https://dependabot.com/docs/config-file/#automerged_updates